### ISSUE TITLE
If redditp is loaded over https, attempt to load all images over https too

### ISSUE THREAD

**Issue by bo0tzz** [ubershmekel/redditp#66]
Currently it seems like redditp will just take the url as it is and load the image. This results in errors such as:
![image](http://i.bo0tzz.me/KYYk7l8t.png)
Would it be worth attempting to load any resources over https first?

**Comment by ubershmekel** [comment]
Yeah, converting all http to https sounds like a good idea. Maybe we need to detect whether redditp is being accessed via https, to be sure we're not causing an issue for intranets.

**Comment by bo0tzz** [comment]
It looks like there is already some code that does this, however if I'm reading correctly it seems like it only replaces http with https if the file being requested is a gif? See [script.js#L322](https://github.com/ubershmekel/redditp/blob/master/js/script.js#L322)

**Comment by ubershmekel** [comment]
Yeah, odd, probably should just throw away those if statements?

**Comment by bo0tzz** [comment]
Seems like it. Possibly replace them with a check for the protocol that redditp was accessed over, like you said earlier?
